queue/job.go

```diff
@@ -43,8 +43,14 @@ type Acknowledger interface {
 // NewJob creates a new Job with default values, a new unique ID and current
 // timestamp.
 func NewJob() *Job {
+	u, err := uuid.NewV4()
+
```

Comment:
I'd remove this empty line between error assignment and error check

Reply:
Changed. I'll squash both commits